ql/src/test/queries/clientpositive/lateral_view_cbo.q

```diff
@@ -0,0 +1,8 @@
+--! qt:dataset:src
+--Q1
+EXPLAIN CBO SELECT myTable.myCol FROM src 
+LATERAL VIEW explode(array(1,2,3)) myTable AS myCol;
```

Comment:
Please include an `EXPLAIN CBO` query with `inline(array(1,2,3)` which seems to be handled differently.

Reply:
Please include an `EXPLAIN CBO` query where the arguments of the table function are not constants:  `explode(table.array_col)`.

Reply:
Done